services/git-service/BareRepository/tests-source/manifests/@tweek/custom_types/version.json

```diff
@@ -10,7 +10,7 @@
         "value": {
 			"base":"string",
 			"comparer":"version",
-			"validation":"/[0-9.]/"
+			"validation":"/[0-9]*.\\.[0-9.]*.\\.[0-9.]*./"
```

Comment:
It matched it, probably because of the `*`, but we can try this:
`[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{0,3}`, assuming version has always 4 parts.